logstash-core/src/test/java/org/logstash/ackedqueue/QueueTestHelpers.java

```diff
@@ -0,0 +1,29 @@
+package org.logstash.ackedqueue;
+
+import org.logstash.ackedqueue.io.ByteBufferPageIO;
+
+import java.io.IOException;
+
+/**
+ * Class containing common methods to help DRY up acked queue tests.
+ */
+public class QueueTestHelpers {
+
+    /**
+     * Returns the minimum capacity required for {@link ByteBufferPageIO}
+     * @return int - minimum capacity required
+     */
+    public static int minimumCapacityForByteBufferPageIO(){
+        return ByteBufferPageIO.WRAPPER_SIZE;
```

Comment:
@robbavey this seems kinda verbose, can't we just make this a `public static final int` constant instead of a function call ? :)